plugin.py

```diff
@@ -18,21 +21,29 @@ class LspPyrightPlugin(NpmClientHandler):
     server_directory = "language-server"
     server_binary_path = os.path.join(server_directory, "node_modules", "pyright", "langserver.index.js")
 
-    @classmethod
-    def additional_variables(cls) -> Optional[Dict[str, str]]:
-        variables = super().additional_variables() or {}
-        variables.update(
-            {
-                "sublime_py_files_dir": os.path.dirname(sublime.__file__),
-            }
-        )
-
-        return variables
-
     @classmethod
     def install_in_cache(cls) -> bool:
         return False
 
     @classmethod
     def minimum_node_version(cls) -> Tuple[int, int, int]:
         return (12, 0, 0)
+
+    @classmethod
+    def on_client_configuration_ready(cls, configuration: Dict[str, Any]) -> None:
+        super().on_client_configuration_ready(configuration)
+
+        if get_setting("st_plugin_development_mode"):
+            configuration["settings"]["python.analysis.extraPaths"].extend(cls.find_package_dependency_dirs())
```

Comment:
```
configuration["settings"]["python.analysis.extraPaths"]
```
this may throw a KeyError if `python.analysis.extraPaths` does not exist for some reason. Better to use the `setdefault` method.

Reply:
Why it would not exist :thinking: 

If we need to give it a default value, then it would be the one in `LSP-pyright.sublime-settings`, then it seems we are doing something unnecessarily duplicate.